app/models/rib.rb

```diff
@@ -10,12 +10,12 @@ class Rib < ApplicationRecord
   scope :reused, -> { Rib.where(iban: Rib.multiple_ibans) }
   scope :multiple_ibans, -> { Rib.select(:iban).group(:iban).having("count(iban) > 1") }
 
-  normalizes :bic, with: ->(bic) { bic.strip }
-  normalizes :iban, with: ->(iban) { iban.strip }
+  normalizes :bic, :iban, with: ->(value) { value.gsub(/\s+/, "").upcase }
+  normalizes :name, with: ->(name) { name.squish }
```

Comment:
Les tests me donnent l'impression qu'on fait le même traitement sur ces deux attributs, mais qu'on utilise 2 méthodes différentes. 

Je ne vois ptet pas la subtilité, mais j'ai l'impression que `.gsub(/\s+/, "")` et `squish` font la même chose.

Reply:
les espaces doivent être préservés à l'intérieur du nom, pas comme les IBAN/BIC (c.f la spec qui a un bon exemple)

Reply:
Ok, got it :+1: